assets/airports/kcvg.json

```diff
@@ -0,0 +1,254 @@
+        "ZEGUL": ["N38d49.74", "W84d35.12"],
+        "ZUGLU": ["N38d55.55", "W84d45.20"]
+    },
+    "runways": [
+        {
+            "name": ["09", "27"],
+            "name_offset": [[0, 0], [0, 0]],
+            "end": [["N39d2.78", "W84d41.71", "883.3ft"], ["N39d2.78", "W84d39.17", "874.9ft"]],
+            "delay": [10, 10],
+            "ils": [true, true]
+        },
+	    {
+            "name": ["18C", "36C"],
+            "name_offset": [[0, 0], [0, 0]],
+            "end": [["N39d3.88", "W84d40.12", "874.6ft"], ["N39d2.07", "W84d40.12", "840.9ft"]],
+            "delay": [10, 10],
+            "ils": [true, true]
+        },
+	    {
+            "name": ["18L", "36R"],
+            "name_offset": [[0, 0], [0, 0]],
+            "end": [["N39d3.35", "W84d38.80", "886.2ft"], ["N39d1.70", "W84d38.81", "896.2ft"]],
+            "delay": [10, 10],
+            "ils": [true, true]
+        },
+	    {
+            "name": ["18R", "36L"],
+            "name_offset": [[0, 0], [0, 0]],
+            "end": [["N39d4.25", "W84d41.02", "865.4ft"], ["N39d2.94", "W84d41.03", "873.4ft"]],
+            "delay": [10, 10],
+            "ils": [true, true]
+        }
+    ],
+    "sids": {
+        "BNGLE4": {
+            "icao": "BNGLE4",
+            "name": "Bngle Four",
+            "rwy": {
+                "18L": ["CAMUM", "ANTEO", "CODKO", "JBETH", "FOPAS"],
+                "18C": [["FLOEE", "A14+"], "ZUGLU", "MIPIE", "HERMM", "AMMOT"],
+                "18R": ["FANSA", "ZUGLU", "MIPIE", "HERMM", "AMMOT"],
+                "27": [["BONLE", "A14+"], "AMMOT"],
+                "36R": [["HAGSS", "A14+"]],
+                "36C": ["CIGMA", "DJOHN"],
+                "36L": ["CIGMA", "DJOHN"]
+            },
+            "body": [["BNGLE", "A80+"]],
+            "exitPoints": {
+                "RIKLE": [["RIKLE", "A180+"]],
+                "DJB": [["RIKLE", "A180+"], "DJB"]
+            },
+            "draw": [
+                ["CAMUM", "ANTEO", "CODKO", "JBETH", "FOPAS", "BNGLE"],
+                ["FLOEE", "ZUGLU"],
+                ["FANSA", "ZUGLU"],
+                ["ZUGLU", "MIPIE", "HERMM", "AMMOT"],
+                ["BONLE", "AMMOT"],
+                ["AMMOT", "BNGLE"],
+                ["HAGSS", "BNGLE"],
+                ["CIGMA", "DJOHN", "BNGLE"],
+                ["BNGLE", "RIKLE*"],
+                ["RIKLE", "DJB*"]
+            ]
+        },
+        "CHCLL4": {
+            "icao": "CHCLL4",
+            "name": "Chcll Four",
+            "rwy": {
+                "18L": ["CAMUM", "ZEGUL"],
+                "18R": ["FANSA"],
+                "18C": [["FLOEE", "A14+"], "FEBLU"],
+                "27": [["BONLE", "A14+"], "CULED"],
+                "36L": ["HOBAL", "BOUMR", "CULED"],
+                "36C": ["HOBAL", "BOUMR", "CULED"],
+                "36R": [["HAGSS", "A14+"], "EVAME", "CEMLA", "FRTZY", "CULED"]
+            },
+            "body": [["CHCLL", "A85+"], ["PONIE", "A104+"]],
```

Comment:
This altitude restriction is not listed on the charts; please remove from all other ones